tasks/tasks.js

```diff
@@ -1,3 +1,5 @@
+const sequelize = require('../utilities/db');
```

Comment:
nit: Can we combine this with the other `requires` right below?

Reply:
Good idea. Made the change.